server/src/Api/Extensions/FormFileExtentions.cs

```diff
@@ -0,0 +1,24 @@
+using Microsoft.AspNetCore.Http;
+using StudyZen.Api.Exceptions;
+using StudyZen.Application.Services; 
+
+namespace StudyZen.Api.Extensions
+{
+    public static class FormFileExtensions
+    {
+        public static void ImportFlashcardsFromCsvStream(this IFormFile file, FlashcardFileImporter importer, int flashcardSetId)
+        {
+            try
+            {
+                using (var stream = file.OpenReadStream())
```

Comment:
Not sure how useful is having this extension. Seems like it could be handled in the service itself.